src/components/HeaderLink.astro

```diff
@@ -14,20 +14,13 @@ const isActive = href === pathname || href === "/" + subpath?.[0];
   href={href}
   class:list={[
     className,
-    { active: isActive },
-    "text-whitee font-Montserrat text-lg font-normal uppercase hover:text-lightyellow",
+    {
+      "font-bold": isActive,
+      "font-normal": !isActive,
```

Comment:
negation of `isActive`, `font-normal` will be applied if link is not active (you are not currently on that page)